OSVR-Unity/Assets/OSVRUnity/src/ClientKit.cs

```diff
@@ -30,6 +30,7 @@ public class ClientKit : MonoBehaviour
             public string AppID;
 
             private OSVR.ClientKit.ClientContext contextObject;
+            private static ClientKit _instance;
```

Comment:
That is automatic. Variables are automatically initialized to a default value: https://msdn.microsoft.com/en-us/library/aa691171(v=vs.71).aspx

So it's a matter of preference/standard. I tend to agree with this post that there is no need to initialize to a default value. http://stackoverflow.com/questions/24551/best-practice-initialize-class-fields-in-constructor-or-at-declaration